spec/widgets/toc.test.js

```diff
@@ -60,7 +60,7 @@ describe('Table of Contents', function() {
       });
 
       expect(this.sandbox.find('.toc')).toExist();
-      expect(this.sandbox.find('h2 span')).toContainText('No index available');
+      expect(this.sandbox.find('h2 span')).toContainText(i18n.t('noIndex'));
```

Comment:
Looks like this test is failing because we merged your other PR with the more up-to-date i18next library. Could you make this update, and perhaps shield you local travis.yml?